bigtable/tableadmin.rb

```diff
@@ -0,0 +1,178 @@
+# frozen_string_literal: true
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Import google bigtable client lib
+require "google-cloud-bigtable"
+
+def run_table_operations instance_id, table_id
+  bigtable = Google::Cloud.new.bigtable
+  p "==> Checking if table exists"
+  table = bigtable.table(instance_id, table_id, perform_lookup: true)
```

Comment:
For samples where variables are defined via method args, could you include a comment describing its function, something like: 
```ruby
# instance_id = "Your Bigtable instance ID"
```